aws/vpc/main.tf

```
@@ -89,7 +89,27 @@ resource "aws_nat_gateway" "nat_gateway" {
 }
 
 /*
- * Routes for private subnets to use NAT gateway
+ * Set use_transit_gateway to true create transit gateway attachments in the private subnets and
+ * route traffic to the TGW instead of NAT GW.
+ * Should be used when create_nat_gateway=false.
+ */
+resource "aws_ec2_transit_gateway_vpc_attachment" "transit_gateway" {
+  count = var.use_transit_gateway ? 1 : 0
+  subnet_ids                                      = aws_subnet.private_subnet.*.id
```

This is a place where terraform is picky about formatting (but will automatically fix it for you, locally).

It either needs a line break between this `count` line and those below it, or the `=` sings must all line up.
```suggestion
  count = var.use_transit_gateway ? 1 : 0

  subnet_ids                                      = aws_subnet.private_subnet.*.id
```